src/firebase/firebaseConfig.js

```diff
@@ -0,0 +1,16 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+  apiKey: "APİ_KEY",
```

Comment:
Dummy data bile olsa typo yapmamak daha iyi olacaktır :)
İngilizce'de büyük "İ" harfi olmadığı için `apiKey: "API_KEY",` yapmak gerekir.